examples/diagram-fit-variants/index.html

```diff
@@ -144,42 +150,22 @@ <h2>Various fit variants</h2>
             }
         }
 
-        // TODO use the lib methods when available, see https://github.com/process-analytics/bpmn-visualization-js/issues/738
         document.getElementById('btn-zoom-actual').onclick = function() {
             fitType = undefined;
-            bpmnVisualization.graph.zoomActual();
+            bpmnVisualization.fit({type: bpmnvisu.FitType.None});
         };
         document.getElementById('btn-fit-horizontally').onclick = function() {
-            performFit('horizontal');
+            bpmnVisualization.fit({type: bpmnvisu.FitType.Horizontal});
         };
         document.getElementById('btn-fit-vertically').onclick = function() {
-            performFit('vertical');
+            bpmnVisualization.fit({type: bpmnvisu.FitType.Vertical});
         };
         document.getElementById('btn-fit-all').onclick = function() {
-            performFit('all');
+            bpmnVisualization.fit({type: bpmnvisu.FitType.HorizontalVertical});
+        };
+        document.getElementById('btn-fit-center').onclick = function() {
+            bpmnVisualization.fit({type: bpmnvisu.FitType.Center});
         };
-
-        function performFit(type) {
-            if (type) {
-                fitType = type;
-            }
```

Comment:
:warning: this breaks the margin management.
Why not keeping this function and update the implementation to use the bpmn-viusalization API? 
this implies less changes and make the margin changes work

Reply:
now fixed